test/helpers.js

```diff
@@ -0,0 +1,12 @@
+const { readFileSync } = require('fs');
+const { resolve } = require('path');
+
+const getRefSchemaUrl = () => {
+  const schemaPath = resolve(__dirname, '../schema.json');
+  const schema = JSON.parse(readFileSync(schemaPath, 'utf-8'));
+  return schema.$id;
```

Comment:
The function returns the value of $id of the local schema file which is `https://schemas.en10204.io/en10168-schemas/v0.0.3/schema.json` so it resolves again to the remote schema.json. Wrote a little helper `getSchema`

Reply:
switched to AJV